packages/client/mutations/EndCheckInMutation.ts

```diff
@@ -0,0 +1,155 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import onMeetingRoute from '~/utils/onMeetingRoute'
+import {EndCheckInMutation_notification} from '~/__generated__/EndCheckInMutation_notification.graphql'
+import {EndCheckInMutation_team} from '~/__generated__/EndCheckInMutation_team.graphql'
+import Atmosphere from '../Atmosphere'
+import {
+  HistoryMaybeLocalHandler,
+  OnNextHandler,
+  OnNextHistoryContext,
+  SharedUpdater,
+  StandardMutation
+} from '../types/relayMutations'
+import {EndCheckInMutation as TEndCheckInMutation} from '../__generated__/EndCheckInMutation.graphql'
+import handleRemoveSuggestedActions from './handlers/handleRemoveSuggestedActions'
+import handleRemoveTasks from './handlers/handleRemoveTasks'
+import handleUpsertTasks from './handlers/handleUpsertTasks'
+import handleAddTimelineEvent from './handlers/handleAddTimelineEvent'
+import {RecordProxy} from 'relay-runtime'
+
+graphql`
+  fragment EndCheckInMutation_team on EndCheckInSuccess {
+    isKill
+    meeting {
+      id
+      endedAt
+      teamId
+      ... on ActionMeeting {
```

Comment:
the meeting is already of type ActionMeeting so this inline fragment isn't necessary